### ISSUE TITLE
The software don't output any image

### ISSUE THREAD

**Issue by TUCYDAbc** [BobJr23/MTGA_Swapper#1]
I have run the software, but it only show all the names of the cards without output any images. Is it expired?

**Comment by BobJr23** [comment]
Yes it seems that they updated their code. I'll update when there's changes to this code

**Comment by TUCYDAbc** [comment]
OK, thanks! By the way, can this program be used to extract the files in "AssetBundle" folder, which contains lots of small .mtga file?

**Comment by BobJr23** [comment]
Fixed
